common/source/docs/common-frsky-repurposed.rst

```diff
@@ -0,0 +1,95 @@
+.. _common-frsky-repurposed:
+
+FrSky Telemetry Using Repurposed Messages
```

Comment:
This separation from common-frsky-mavlink.rst is confusing.
This information should be together with the mentioned page